src/handlers.js

```diff
@@ -266,6 +340,82 @@ async function statusHandler(message) {
   }
 }
 
+/**
+ * Post updates to GitHub, when the status of a task changes. Uses Checks API
+ **/
+async function statusHandler(message) {
+  let {taskGroupId, state, runs, taskId} = message.payload.status;
+  let {runId} = message.payload;
+  let {reasonResolved} = runs[runId];
+
+  let build = await this.context.Builds.load({
+    taskGroupId,
+  });
+
+  let {organization, repository, sha, eventId, eventType, installationId} = build;
+
+  let debug = Debug(`${debugPrefix}:${eventId}`);
+  debug(`Handling state change for task ${taskId} in group ${taskGroupId}`);
+
+  let taskState = {
+    status: 'completed',
+    conclusion: CONCLUSIONS[reasonResolved || state],
```

Comment:
Maybe we need to have a case here for if neither are already defined? Default to failure I expect. This could be the case if we add a new `reason` to exceptions and don't update here for instance. Another option would be to throw an error on our end so that we fix it.

Reply:
state is a required prop on these exchange messages

Reply:
oh, you mean if there's a new element in the `enum`. Yes, that's a great idea! Why default to failure tho? What if that is not a failure type of reason, but somebody PR would be blocked? Maybe `Action required`? As in, "hey, we got an unknown message please click the link and check it out". Or `neutral`, with the reason in the description of the checkrun

Reply:
Sure, both sound good to me. Also log something that we can alert on later!

Reply:
oh, log something - good point!